app/login/page.tsx

```diff
@@ -103,7 +103,9 @@ const LoginPage = () => {
               Login
             </Button>
             <div className="flex justify-center">
-              <Button className="mt-4">
+              <Button className="mt-4" onClick={async ()=>{
```

Comment:
Avoid directly calling google login using callback rather define a function and use it here.